RecoVertex/BeamSpotProducer/test/alcareco.py

```diff
@@ -65,7 +65,8 @@
 #process.GlobalTag.globaltag = 'IDEAL_V9::All'  # take your favourite
 
 process.GlobalTag.connect = "sqlite_file:/afs/cern.ch/user/f/futyand/public/globaltag/10PB_21X.db"
-process.GlobalTag.globaltag = '10PB_V1::All'
+#process.GlobalTag.globaltag = '10PB_V1::All'
+process.GlobalTag.globaltag = cms.string( autoCond[ 'phase1_2022_realistic' ] )
```

Comment:
@francescobrivio is this config for MC or data, do you maybe know?

Reply:
I have never used it and as you can see it was last touched in 2013 😅 , but from a quick look it seems to me it can actually be used for both data or MC. So let's put a data GT (the Prompt one I'd suggest) and add a comment just afte: `# pick your favourite`

Reply:
but @francescobrivio if it doesnt matter we should just go with what the PR author picked, no?

Reply:
All of the files should either be made unit tests or deleted..